### DISCUSSION TITLE
Feature Request: adding a back to top button in homepage

### DISCUSSION THREAD

**Discussion by chinmay7016** [TheOdinProject/theodinproject#4252] [Ideas]
### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] The title of this issue follows the `Feature Request: brief description of feature request` format, e.g. `Feature Request: Add a dark mode to the website`
- [X] Would you like to work on this issue?

### Description of the Feature Request

When I visit a website I wanna add back to top button on homeoage

### Acceptance criteria

-[ ] button
-[ ] arrow

### Additional Comments

Can you pls tell which file to edit

**Comment by CouchofTomato** [comment]
@KevinMulhern Any thoughts on this please mate?

**Comment by KevinMulhern** [comment]
I'm not sure if we need it, our home page isn't that large. I'll move this to a discussions feature request.
